src/Network/Network.Test/ScenarioTests/AzureFirewallTests.cs

```diff
@@ -34,31 +34,31 @@ public void TestAzureFirewallCRUD()
             TestRunner.RunTestScript("Test-AzureFirewallCRUD");
         }
 
-        [Fact]
+        [Fact(Skip = "skip")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         [Trait(Category.Owner, NrpTeamAlias.azurefirewall)]
         public void TestAzureFirewallCRUDWithZones()
         {
             TestRunner.RunTestScript("Test-AzureFirewallCRUDWithZones");
         }
 
-        [Fact]
+        [Fact(Skip ="skip")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         [Trait(Category.Owner, NrpTeamAlias.azurefirewall)]
         public void TestAzureFirewallPIPAndVNETObjectTypeParams()
         {
             TestRunner.RunTestScript("Test-AzureFirewallPIPAndVNETObjectTypeParams");
         }
 
-        [Fact]
+        [Fact(Skip = "skip")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         [Trait(Category.Owner, NrpTeamAlias.azurefirewall)]
         public void TestAzureFirewallAllocateAndDeallocate()
         {
             TestRunner.RunTestScript("Test-AzureFirewallAllocateAndDeallocate");
         }
 
-        [Fact]
+        [Fact(Skip = "skip")]
```

Comment:
Why do you want to skip these test cases?

Reply:
That was just for testing. Have reverted this change.